src/controller/securityhub/controller.go

```diff
@@ -63,12 +64,19 @@ func WithArtifact(enable bool) Option {
 type Controller interface {
 	// SecuritySummary returns the security summary of the specified project.
 	SecuritySummary(ctx context.Context, projectID int64, options ...Option) (*secHubModel.Summary, error)
+	// ListVuls list vulnerabilities by query
+	ListVuls(ctx context.Context, scannerUUID string, projectID int64, query *q.Query) ([]*secHubModel.VulnerabilityItem, error)
+	// CountVuls get all vulnerability count by query
+	CountVuls(ctx context.Context, scannerUUID string, projectID int64, tuneCount bool, query *q.Query) (int64, error)
+	// DefaultScannerUUID get default scanner UUID
+	DefaultScannerUUID(ctx context.Context) (string, error)
```

Comment:
we should put the DefaultScannerUUID in the scannerMgr

Reply:
Updated